lib/y2logsstep.pm

```diff
@@ -164,6 +165,15 @@ sub deal_with_dependency_issues {
     }
 }
 
+sub sle15_workaround_broken_patterns {
+    return unless sle_version_at_least('15');
+    # SLE 15 has pattern errors, workaround them - rbrown 04/07/2017
+    while (check_screen('sle-15-failed-to-select-pattern')) {
```

Comment:
why did you change the timeout? This will now cause a lot of waiting time, 30 seconds default. Do you think less could work?